docs/view_components/welcome.rst

```diff
@@ -0,0 +1,22 @@
+************
+Welcome page
+************
+
+After analyzing the post-Q3 meeting inputs regarding the tool interface design, all but one design have a welcome message as part of the
```

Comment:
No need to mention Q3 meeting, this is meant to be the documentation for future users of the interface